### DISCUSSION TITLE
PS3 controller connection issues

### DISCUSSION THREAD

**Discussion by AtumRa5** [darthcloud/BlueRetro#715] [Q&A]
I'm having a problem where, after doing everything right to install the correct firmware and input my ps3 controller's mac address to connect to the blue retro device, the controller always connects to the ps2 as "controller number 3", and i can't change the ps3 controller number 3 to 1, so I can't use the controller on the ps2, it pairs to blue retro but I can't control the ps2.

**Comment by pablopsw** [comment]
most likely you have installed the wrong .bin, remember that there is a version for internal and external installation, try the other one.

**Reply by AtumRa5** [reply]
> most likely you have installed the wrong .bin, remember that there is a version for internal and external installation, try the other one.

Internal and external installation?
I tried the 1.8.3 version using playstation.bin, I also tried version 1.8.2, and tried the universal bin and playstation bin. I also installed sixaxis, setted the right mac address of the controller to the blue retro device, I dont know what am I doing wrong, I tried everything according to the video tutorials I watched on youtube and the official tutorial on this github. I'm always getting the same result, the same error trying different stuff, I'm starting to think the problem is on the hardware of the blue retro I bought, not the software. My PS3 controller is original, I have others here I tried as well and got the same result. It does connect to the blue retro, but as controller number 3, and when I try using it, nothing works.

**Reply by pablopsw** [reply]
I also followed tutorials and something similar happened to me, only it marked me as command 2, in my case it was an internal installation, but when using the internal .bin file, it didn't work, I tried the external installation one and it worked. I hope you can solve it.
